dspy/primitives/example.py

```diff
@@ -105,3 +105,17 @@ def without(self, *keys):
     
     def toDict(self):
         return self._store.copy()
+    
```

Comment:
This is great!

Reply:
Ideally, we'd need a better paradigm for serialization. Manually adding all the dump and load information seems cumbersome.